### DISCUSSION TITLE
AttributeError: module 'autokeras' has no attribute 'ImageClassifier'

### DISCUSSION THREAD

**Discussion by liuamo** [keras-team/autokeras#1414] [Q&A]
Why AttributeError: module 'autokeras' has no attribute 'ImageClassifier'

**Comment by haifeng-jin** [comment]
This is probably because your python path is using the autokeras directory under tests/autokeras as the module. Are you using pycharm?

**Reply by SwiftWinds** [reply]
Yep, turned out my file was called `autokeras.py` 🤦‍♂️ thanks for the help
